cmd/puppet-db/root_test.go

```diff
@@ -0,0 +1,30 @@
+package main
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/spf13/viper"
+	"github.com/stretchr/testify/assert"
+)
+
+func fixturePath() string {
+	path, err := os.Getwd()
+	if err != nil {
+		panic(err)
+	}
+	return filepath.Join(filepath.Dir(filepath.Dir(path)), "acceptance", "fixtures")
```

Comment:
I'm not sure about having UT fixtures in acceptance dir, in `puppet-code` we had a `testing` dir in places where we needed additional files(mocks)

Reply:
I created the `testdata` directory, same as puppet-code.